compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -2619,13 +2619,15 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
             case sym if sym.isType => sym.asType
           }.toList
 
-        def memberType(name: String): Symbol = typeMember(name)
-        def typeMember(name: String): Symbol =
+        def memberType(name: String): Symbol =
           self.unforcedDecls.find(sym => sym.name == name.toTypeName)
```

Comment:
I guess this was present before, but any reason we use unforcedDecls here? That could also miss declarations.

Reply:
I do not belive there was a particular reason. Which method would you use instead?

Reply:
`decls`

Reply:
done